bokeh/models/annotations.py

```diff
@@ -40,10 +40,16 @@ class Legend(Annotation):
     The %s for the legend background style.
     """)
 
+    background_fill_color = Override(default="#ffffff")
+
     label_props = Include(TextProps, help="""
     The %s for the legend labels.
     """)
 
+    label_text_baseline = Override(default='middle')
+
+    label_text_font_size = Override(default={ 'value' : '10pt' })
+
```

Comment:
it's overriding the default for a prop created by Include. It would also work to specify the prop from scratch but would have to repeat the type and help, while this way we get the type and help from the Include.